linkerd/core/src/main/scala/io/buoyant/linkerd/LoadBalancerConfig.scala

```diff
@@ -16,8 +17,11 @@ import com.twitter.finagle.loadbalancer.{Balancers, LoadBalancerFactory}
 trait LoadBalancerConfig {
   val factory: LoadBalancerFactory
 
+  val enableProbation: Option[Boolean] = None
+
   @JsonIgnore
-  def clientParams = Stack.Params.empty + LoadBalancerFactory.Param(factory)
+  def clientParams = Stack.Params.empty + LoadBalancerFactory.Param(factory) +
+    LoadBalancerFactory.EnableProbation(enableProbation.getOrElse(true))
```

Comment:
I was trying to be explicit about defaulting to true, but I am totally open to changing it.